backend/src/providers/logs/cloudwatch.provider.ts

```
@@ -530,4 +528,190 @@ export class CloudWatchProvider extends BaseLogProvider {
+          user_agent: userAgent,
+          event_message: requestLine,
+          metadata: existingMeta,
+        };
+      }
+
+      // Application logs: prefix the level so the dashboard message column
+      // reads `info - some message`, matching Vector's
+      // `join!([req.level, req.message], " - ")`. Leaves `error`/`stack` as
+      // top-level keys in the body for the detail panel.
+      const eventMessage =
+        level !== undefined && msgField ? `${level} - ${msgField}` : msgField;
```

ESLint

[error] Delete `⏎·······`

(prettier/prettier)